owncloud/web-extensions
# feat: add nyan cat progress bar extension

- brings dependencies in sync with web repo
- adds a NyanCat style progress bar for the global progress bar extension point, see screen capture below
- adds manifest.json files for both apps (cast and progress-bars), as described in https://owncloud.dev/services/web/#application-structure

https://github.com/user-attachments/assets/a87ca7ea-bc31-4c9e-b628-9c0306f90694


## how to test this manually

1. `pnpm install`
2. `cd packages/web-app-progress-bars`
3. `pnpm build`
4. copy the resulting `dist` folder (it should contain a `progress-bars.js` and a `manifest.json` file) into your web repo dev stack, e.g. as `web/dev/docker/apps/progress-bars` (i.e. rename the `dist` folder to something more telling)
5. (re-)start ocis
